src/cluster_legacy.c

```diff
@@ -2971,7 +2971,7 @@ int clusterProcessPacket(clusterLink *link) {
                 /* If the reply has a non matching node ID we
                  * disconnect this node and set it as not having an associated
                  * address. */
-                serverLog(LL_DEBUG,
+                serverLog(LL_WARNING,
```

Comment:
Downgrade to NOTICE? My understanding is that WARNING is reserved for admin actionable failures, as suggested by @madolson.

Reply:
NOTICE sounds good to me. Let's use that and merge this. :)